Assets/scss/components/_profile-card.scss

```diff
@@ -0,0 +1,101 @@
+.cmp-profile-card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $white-smoke;
+    margin: 0;
+
+    &__container {
+        display: flex;
+        flex-wrap: wrap;
+        gap: 20px;
+        justify-content: center;
+        padding: 20px;
+
+        &__profile {
+            width: 300px;
+            border: 1px solid #ccc;
+            border-radius: 10px;
+            overflow: hidden;
+            text-align: center;
+            box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+            font-family: $arial;
+            background-color: $white;
+            margin: 20px;
+            &:hover {
+                background-color: $kingly-cloud;
+            }
+
+            &.light {
+                background-color: var(--profile-card-light-background-color);
+                &:hover {
+                    background-color: $dark-orange;
+                }
+            }
+
+            &.dark {
+                background-color: var(--profile-card-dark-background-color);
+                &:hover {
+                    background-color: $burlywood;
+                }
+            }
+
+            &__image {
+                width: 150px;
+                height: 150px;
+                margin: 8px auto 0;
+                border-radius: 50%;
+                overflow: hidden;
+
+                img {
+                    height: 100%;
+                    object-fit: cover;
+                }
+            }
+
+            &__details {
+                padding: 20px;
+            }
+
+            &__name {
+                margin: 0;
+                font-size: 24px;
+                color: $dark-orange;
+
+                .light & {
+                    color: $sonic-silver;
+                }
+            }
+
+            &__title,
+            &__company {
+                margin: 5px 0;
+                font-size: 16px;
```

Comment:
All font-sizes could use `toRem()` function

Reply:
I have done the changes.